ProQuest.js

```diff
@@ -449,13 +471,15 @@ function scrape(doc, url, type, pdfUrl) {
 			mimeType: 'application/pdf'
 		});
 	} else {
-		var pdfLink = ZU.xpath(doc, '(//div[@id="side_panel"]//\
+		var pdfLink = ZU.xpath(doc, '//li[@id="tab-SPDF-null" or @id="tab-FulltextPreview-null"]/a');
```

Comment:
I deleted the else-case and consecutively the function `fetchEmbeddedPdf` is not called anymore, i.e. we could delete that as well. Okay?